### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by HeliaHashemipour** [lucidrains/denoising-diffusion-pytorch#297]
Hi, 
Can you tell me what is 
train_num_steps and timesteps?
I can't get the difference between them. (which one of them is the timesteps in the paper?)

**Comment by dancingacidpanda** [comment]
train_num_steps is the number of times the gradient is computed (for every train_batch_size * gradient_accumulate_every samples in your dataset) until the model stops training

timesteps is "the one from the paper"
